src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private string GetActivationCodeFromDb(string userEmail)
+        {
+            return "pttigkgkgkgkgk6666k6k6k6k6k6k6k1123ndncj";
```

Comment:
Whats going on here? You have a method that creates a unique code, why is this still here then?

Reply:
Seen. Forgot to replace the test value after implementing the method. Fixing now.

Reply:
The method to generate a unique code doesn't apply to my send mail task. My methods here are designed to send a mail to an already existing user. The code in the return statement is representing a user's code from the db.

Am I wrong?

Reply:
We never agreed to get code from the DB naa.

Reply:
Ok. I must've assumed wrong. I'll call the method instead.